evap/staff/templates/staff_semester_view.html

```diff
@@ -371,7 +371,7 @@ <h3>
                             <col />
                             <col style="width: 14em" />
                             <col style="width: 12em" />
-                            <col style="width: 9em" />
+                            <col style="width: 12.9em" />
```

Comment:
@janno42 do we have a mechanic to make a column "shrink-to-fit"?

Reply:
@FranzSw is working on the first implementation in #1889